src/Facets/GetGasFacet.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
```

Comment:
There is no .md file for the new smart contract. 
It should be added to have a complete documentation.